Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -361,4 +419,33 @@ public class CRTClientEngine: HTTPClient {
         response.body = .stream(stream)
         return requestOptions
     }
+
+    actor ContinuationFlag {
+        var continuationResumed = false
+        public func setFlag(val: Bool) {
```

Comment:
nit: rename to something more descriptive like `setContinuationResumed` and since `set` is the name we can remove val: when using the function

ex. `public func setContinuationResumed(_ val: Bool)`
usage: `public func setContinuationResume(true)`

Reply:
Changed as suggested.